app/build.gradle

```diff
@@ -161,6 +161,8 @@ def requireWifi = project.requireWifi ?: 'false';
 // database unencrypted so that it can be easily inspected using Stetho.
 def encryptionPassword = project.encryptionPassword ?: '';
 
+def syncAccountName = "sync";
```

Comment:
Is there a reason for defining this in the `build.gradle` instead of in the code where it was defined before?